.github/workflows/publish-docker-hub.yaml

```diff
@@ -15,5 +15,5 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
           repository: unitytechnologies/datasetinsights
-          tags: latest
+          tags: 1.0
```

Comment:
Why do we want to tag this as 1.0 instead of latest?

Reply:
- latest tag should have model experimentation code until experimentation repo is up and ready. So we don't want latest tag to be created when we release code from master.

Reply:
I will recommend leave it as latest. Users can still access to docker image with tag 0.2.5, 0.2.6,.... and I see no harm having the `latest` branch always reflect the current master.